pkg/stable.packages.props

```diff
@@ -0,0 +1,950 @@
+      <Version>1.0.0</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.ubuntu.16.04-x64.Microsoft.NETCore.TestHost">
+      <Version>1.0.0</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win7-arm.Microsoft.NETCore.TestHost">
+      <Version>1.0.0</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win7-x64.Microsoft.NETCore.TestHost">
+      <Version>1.0.0</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win7-x86.Microsoft.NETCore.TestHost">
+      <Version>1.0.0</Version>
+    </StablePackage>
+    <StablePackage Include="Microsoft.NETCore.Windows.ApiSets">
+      <Version>1.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win7-x64.Microsoft.NETCore.Windows.ApiSets">
+      <Version>1.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win7-x86.Microsoft.NETCore.Windows.ApiSets">
+      <Version>1.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win8-x64.Microsoft.NETCore.Windows.ApiSets">
+      <Version>1.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win8-x86.Microsoft.NETCore.Windows.ApiSets">
+      <Version>1.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win81-x64.Microsoft.NETCore.Windows.ApiSets">
+      <Version>1.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win81-x86.Microsoft.NETCore.Windows.ApiSets">
+      <Version>1.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win10-arm-aot.runtime.native.System.IO.Compression">
+      <Version>4.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win10-x64-aot.runtime.native.System.IO.Compression">
+      <Version>4.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win10-x86-aot.runtime.native.System.IO.Compression">
+      <Version>4.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.win8-arm.Microsoft.NETCore.Runtime.CoreCLR">
+      <Version>1.0.2</Version>
+    </StablePackage>
+
+    <!-- NETCore 1.0 WCF -->
+    <StablePackage Include="System.Private.ServiceModel">
+      <Version>4.1.0</Version>
+    </StablePackage>
+    <StablePackage Include="System.ServiceModel.Duplex">
+      <Version>4.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="System.ServiceModel.Http">
+      <Version>4.1.0</Version>
+    </StablePackage>
+    <StablePackage Include="System.ServiceModel.NetTcp">
+      <Version>4.1.0</Version>
+    </StablePackage>
+    <StablePackage Include="System.ServiceModel.Primitives">
+      <Version>4.1.0</Version>
+    </StablePackage>
+    <StablePackage Include="System.ServiceModel.Security">
+      <Version>4.0.1</Version>
+    </StablePackage>
+    
+    <!-- NETCore 1.0 CoreCLR -->
+    <StablePackage Include="Microsoft.NETCore.Runtime.CoreCLR">
+      <Version>1.0.2</Version>
+    </StablePackage>
+    
```

Comment:
For `dotnet/core-setup` and `dotnet/cli`, I changed over to read the Latest_Packages.txt file, parse it into "DependencyInfo" objects, and we do the full replacement of the version for any packages that match the ID.  It works really well. When NETStandard.Library switched from `1.5` to `1.6` we didn't even notice.  The tool took care of it.

See https://github.com/dotnet/core-setup/blob/master/build_projects/update-dependencies/UpdateFilesTargets.cs#L41-L59

and 

https://github.com/dotnet/core-setup/blob/master/build_projects/update-dependencies/UpdateFilesTargets.cs#L136-L164